### PR TITLE
bank with generics

### PR DESCRIPTION
**Pull request by Yanetta**
bank with generics

### CODE REVIEW

File: src/main/java/bankWithGenerics/BankInstrument.java [hunk 1]
```diff
@@ -1,4 +1,48 @@
 package bankWithGenerics;
 
-public class BankInstrument {
+public abstract class BankInstrument implements Asset {
+    String nameOfInstrument;
```
Comment (SerhiiPasko) [inline src/main/java/bankWithGenerics/BankInstrument.java:4]: private

File: src/main/java/bankWithGenerics/Bank.java [hunk 2]
```diff
@@ -1,4 +1,39 @@
 package bankWithGenerics;
 
-public class Bank {
+public class Bank <T extends Asset> {
+    String bankName;
```
Comment (SerhiiPasko) [inline src/main/java/bankWithGenerics/Bank.java:4]: Fields in clean code are private

File: src/main/java/bankWithGenerics/BankUse.java [hunk 3]
```diff
@@ -1,4 +1,26 @@
 package bankWithGenerics;
 
 public class BankUse {
+    public static void main(String[] args) {
+
+        UsDollar dollar = new UsDollar(300);
+        Bank<UsDollar> usDollarBank = new Bank<UsDollar>("USdollarBank", "Hreschatyk, 1", dollar);
```
Comment (SerhiiPasko) [inline src/main/java/bankWithGenerics/BankUse.java:7]: For same operation use separete methods. You many time repeat same code

Comment (SerhiiPasko) [reply src/main/java/bankWithGenerics/BankUse.java:7]: Or use collections

**Review by SerhiiPasko — APPROVED** [review]
Read comments. Good

**Review by Taniann — APPROVED** [review]
good

**Pull request merged**